### ISSUE TITLE
Add or document option for administrators to register new user accounts

### ISSUE THREAD

**Issue by hoyle-ub** [ncbo/bioportal_web_ui#157]
In addition to the option to disable self-registration, it would be useful to have a means by which administrators can register new user accounts.  I can't find any documentation for this, nor any link to an account creation page in the web UI.

**Comment by hoyle-ub** [comment]
Sorry, I think I was unclear here.  What I’m looking for is a way for an admin to register new users when self-registration is disabled as requested in issue 156 and the sign up link is no longer present.

--
________________________________________________________________________
Patrick Hoyle
Instructional Support Specialist, CASet
College of Arts and Sciences
University at Buffalo
(716) 645-5638

**Comment by graybeal** [comment]
right, so that would be a matter of a script that makes the appropriate call, e.g., through the Ruby console. Since we haven't needed it for BioPortal I'm thinking you might want to develop this yourself. 

Another approach to address both issues could be to only allow users with admin privileges to access the registration page. (You should be able to see an example of how this works on the Admin page, which is only accessible to admins.)

**Comment by jvendetti** [comment]
@hoyle-ub - even if the Sign Up link isn't present, you should be able to go directly to the account creation form using the URL pattern /accounts/new. In the production instance of BioPortal, the URL is: https://bioportal.bioontology.org/accounts/new.
